quasar-utils/src/main/groovy/net/corda/plugins/quasar/QuasarPlugin.groovy

```diff
@@ -25,6 +29,7 @@ class QuasarPlugin implements Plugin<Project> {
 
     private static final String QUASAR = "quasar"
     private static final String MINIMUM_GRADLE_VERSION = "5.1"
+    private static final String JAVAC_PLUGIN = "javac_plugin"
```

Comment:
_Every_ Java Compiler plugin could be called `javac_plugin`. What is so special about this one?

Reply:
I'll change that

Reply:
removed it completely